mmagic/datasets/textual_inversion_dataset.py

```diff
@@ -8,85 +9,39 @@
 
 from mmagic.registry import DATASETS
 
-imagenet_templates_small = [
-    'a photo of a {}',
-    'a rendering of a {}',
-    'a cropped photo of the {}',
-    'the photo of a {}',
-    'a photo of a clean {}',
-    'a photo of a dirty {}',
-    'a dark photo of the {}',
-    'a photo of my {}',
-    'a photo of the cool {}',
-    'a close-up photo of a {}',
-    'a bright photo of the {}',
-    'a cropped photo of a {}',
-    'a photo of the {}',
-    'a good photo of the {}',
-    'a photo of one {}',
-    'a close-up photo of the {}',
-    'a rendition of the {}',
-    'a photo of the clean {}',
-    'a rendition of a {}',
-    'a photo of a nice {}',
-    'a good photo of a {}',
-    'a photo of the nice {}',
-    'a photo of the small {}',
-    'a photo of the weird {}',
-    'a photo of the large {}',
-    'a photo of a cool {}',
-    'a photo of a small {}',
-]
-
```

Comment:
looks like 'imagenet_style_templates_small' is removed, please make sure original textual inversion code runs good as before.

Reply:
Now both imagenet_style_templates_small and imagenet_templates_small are saved as txt files and to be downloaded. Instructions are added to textual_inversion/[README](configs/textual_inversion/README.md).